lib/droplet_kit/models/kubernetes_options.rb

```diff
@@ -0,0 +1,7 @@
+module DropletKit
+  class KubernetesOptions < BaseModel
+
+    attribute :versions
```

Comment:
the options endpoint now also contains `regions` and `sizes` information in addition to the available versions, so it would be good to get those in before merging, or we can just follow up with another PR right after

Reply:
added